pybossa/view/projects.py

```diff
@@ -2925,3 +2925,34 @@ def assign_users(short_name):
 
     flash(msg, 'success')
     return redirect_content_type(url_for('.settings', short_name=project.short_name))
+
+
+@blueprint.route('/<short_name>/answerfieldsconfig', methods=['GET', 'POST'])
+@login_required
+@admin_or_subadmin_required
+def answerfieldsconfig(short_name):
+    """Returns Project Stats"""
+    project, owner, ps = project_by_shortname(short_name)
+    pro = pro_features()
+    ensure_authorized_to('update', project)
+
+    if request.method == 'POST':
+        try:
+            project.info['answer_fields'] = json.loads(request.data)
+            project_repo.save(project)
+            flash(gettext('Configuration updated successfully'), 'success')
+        except Exception:
+            flash(gettext('An error occurred.'), 'error')
+
+    project_sanitized, owner_sanitized = sanitize_project_owner(
+        project, owner, current_user, ps)
+    gold_fields = project.info.get('answer_fields', {})
```

Comment:
to be consistent, can this be named as answer_fields?

Reply:
will fix